nodes/mod.go

```diff
@@ -288,3 +288,33 @@ func downloadModVersion(c echo.Context) error {
 
 	return c.Redirect(302, storage.GenerateDownloadLink(version.Key))
 }
+
+func downloadModVersionArch(c echo.Context) error {
+	modID := c.Param("modId")
+	versionID := c.Param("versionId")
+	platformType := c.Param("platform")
+
+	mod := postgres.GetModByID(c.Request().Context(), modID)
+
+	if mod == nil {
+		return c.String(404, "mod not found")
+	}
+
+	version := postgres.GetModVersion(c.Request().Context(), mod.ID, versionID)
+
+	if version == nil {
+		return c.String(404, "version not found")
+	}
+
+	platform := postgres.GetModArchByPlatform(c.Request().Context(), versionID, platformType)
+
+	if platform == nil {
+		return c.String(404, "platform not found")
+	}
```

Comment:
Would there be any benefit to including the modId, versionId, and platformType in the error messages that are returned?

Reply:
I don't see a problem with it, but bear in mind, I did copy from downloadModVersion. I have the change ready if you desire me to commit/push it with both of them changed.

Reply:
https://github.com/satisfactorymodding/smr-api/pull/12/commits/5e5146206a7fe78acf4f87620417a535e6a9a579